examples/hif2a/fit_to_multiple_rbfes.py

```diff
@@ -45,9 +57,9 @@
 )
 
 intermediate_configuration = Configuration(
-    num_gpus=NUM_GPUS,
-    num_complex_windows=30,
-    num_solvent_windows=30,
+    num_gpus=10,
```

Comment:
I think we can keep this as is for now but @badisa will later on need to change this for the gRPC client.

Reply:
Oops, this was a mistake when resolving merge conflicts in https://github.com/proteneer/timemachine/commit/b71f4d28cce4eecf1cb9060ca05fb585401aa651 -- will revert

Reply:
Resolved in https://github.com/proteneer/timemachine/pull/360/commits/573747ab81b782756cae19f4171220a36dd1ba57